web/src/frontend/src/nav-soknad/components/validering/feilUtils.ts

```diff
@@ -0,0 +1,18 @@
+import { Valideringsfeil } from "../../validering/types";
+
+/** Filtrerer slik at det kun er en feilmelding per faktum */
+export const filtrerFeilmeldinger = (
+	feilmeldinger: Valideringsfeil[]
+): Valideringsfeil[] => {
+	if (!feilmeldinger) {
+		return [];
+	}
+
+	return feilmeldinger.reduce(
```

Comment:
Den her tok lang tid å skjønne, ass.
Men kan man tenke at et faktum bare bør kunne ha 1 feil? Sånn, vi viser uansett bare en feilmelding, kan ikke en komponent returnere første ting som feiler i stedet for flere?

Reply:
Fant den på nettet :) . Tenkte det er bedre at den funksjonaliten ligger sentralt, enn at en må gjøre det i hver enkelt komponent.